### ISSUE TITLE
runSimulations.py takes too long when it didn't before

### ISSUE THREAD

**Issue by Mlvluu** [cRz-Shadows/Pokemon_Trainer_Tournament_Simulator#6]
![image](https://github.com/cRz-Shadows/Pokemon_Trainer_Tournament_Simulator/assets/36082336/234cffd5-15e1-43b7-9bf4-5eb1d50c2ec0)

**Comment by cRz-Shadows** [comment]
If it stops like that, there are errors occurring which have probably been caused by your defined teams

**Comment by Mlvluu** [comment]
Do trainer names have a character limit?

**Comment by cRz-Shadows** [comment]
Nope, it will be with the teams themselves. look at the other issue threads I know there was a similar one.
